tutorials/03_physics_plugins.md

```diff
@@ -1,8 +1,8 @@
-\page physicsplugin Understanding the Physics Plugin
+\page physicsplugins Understand Physics Plugin
```

Comment:
Note that changing the page name also changes the URL of the tutorial. Right now, the URL is https://ignitionrobotics.org/api/physics/3.1/physicsplugin.html. If we change them, anyone who had the old links on a website for example, will be left with broken links. So often it is better to stay with "bad" links that work for previous users, than to improve the URLs but break links.

These tutorials are relatively new though, so I think it's not too late to change them. I'll leave it up to you.